lib/plugin.zsh

```diff
@@ -72,7 +72,7 @@ geometry_plugin_register() {
     return 1
   fi
 
-  if geometry_prompt_${plugin#+}_setup $ctx; then
+  if $plugin_setup_function $ctx; then
```

Comment:
Isn't this sanitized on [this](https://github.com/geometry-zsh/geometry/pull/194/files#diff-fa15ebb02da94453c62e6471145db51aR69) line?

Reply:
this line isn't the fix, this was just to reduce redundancy since we have access to the local anyway

Reply:
oh I see now! :+1: